roles/openshift_excluder/tasks/install.yml

```diff
@@ -2,15 +2,15 @@
 - name: Install latest excluder
   package:
     name: "{{ openshift.common.service_type }}-excluder"
-    state: latest
+    state: "{{ excluder_package_state }}"
   when:
   - openshift_excluder_enabled | default(false) | bool
   - not openshift.common.is_containerized | bool
 
 - name: Install latest docker excluder
   package:
     name: "{{ openshift.common.service_type }}-excluder"
-    state: latest
+    state: "{{ docker_excluder_package_state }}"
   when:
   - docker_excluder_enabled | default(false) | bool
   - not openshift.common.is_containerized | bool
```

Comment:
You could leave `state: latest` and rename this task to `tasks/upgrade.yml` and then just have the install call into `status.yml`.

Then you could update the upgrade to use `tasks_from: upgrade` to upgrade the packages prior to enabling/disabling.